src/BuiltInTools/dotnet-watch/LaunchSettingsProfile.cs

```diff
@@ -83,6 +83,12 @@ internal sealed class LaunchSettingsProfile
             }
 
             var defaultProfileKey = launchSettings.Profiles.FirstOrDefault(f => f.Value.CommandName == "Project").Key;
+            if (defaultProfileKey is null)
```

Comment:
if there is no Command named Project, I think it will throw on line 85 with ".Key".  Am I reading it right?

Reply:
It won't since FirstOrDefault returns KeyValuePair

Reply:
In other words, Default for the "Profiles" collection is non-null?

Reply:
It's a dictionary, we are enumerating KVPs, which is struct.